src/App.java

```diff
@@ -1,31 +1,262 @@
+                userWantsToStop = true;
+
+                System.out.println(returnColoredText("green", "│") + " Thanks for playing!");
+            }
+        }
+
+        scan.close();
     }
-    
+
     /* Here you should create all missing methods used by the start() method
      * I have created the signature for ONE of the methods below.
      * You can create any other methods you need as well!
-     */ 
-    
+    */
+
+    /*
+        This is a helper function that will print a prompt to the user
+        and return their input
+
+        Parameters:
+            - scan (type Scanner): the class-wide object for
+            getting user input
+
+            - prompt (type String): the message to display to the user
+
+        Returns:
+            - (type String): the user's input
+    */
+    public String getUserInput() {
+
+        System.out.print(returnColoredText("green", "│ ↳") + " ");
+
+        String input = scan.nextLine();
+
+        return input;
+    }
+
+    /*
+        This is a helper function to return the ANSI color code of a specific color
+
+        Parameters:
+            - colorName (type String): the color name for the function to return
+              the color code for
+
+        Returns:
+            - (type String): the ANSI color code
+    */
+    public String getColorCodes(String colorName) {
+
+        // https://stackoverflow.com/questions/8261075/adding-multiple-entries-to-a-hashmap-at-once-in-one-statement
+        // https://stackoverflow.com/questions/5762491/how-to-print-color-in-console-using-system-out-println
+        HashMap<String, String> colors = new HashMap<String, String>()
+        {{
+            //put("red", "\u001B[31m");
+            //put("yellow", "\u001B[33m");
+            //put("blue", "\u001B[34m");
+            //put("purple", "\u001B[35m");
+            //put("cyan", "\u001B[36m");
+            //put("white", "\u001B[37m");
+            put("green", "\u001B[32m");
+        }};
+
+        return colors.get(colorName);
+    }
+
+    /*
+        This is a helper function to print line to console in a specific color
+
+        Parameters:
+            - colorName (type String): the color name for the function to print
+              in
+            - message (type String): the message to display to the user
+
+        Returns:
+            - void
+    */
+    public void printColoredLn(String colorName, String message) {
+        System.out.println(getColorCodes(colorName) + message + ANSI_RESET);
+    }
+
+    /*
+        This is a helper function to return a string in a specific color
+
+        Parameters:
+            - colorName (type String): the color name for the function to return
+              the color text for
+            - message (type String): the message to embed in formatted color codes
+
+        Returns:
+            - (type String): the colored text
+    */
+    public String returnColoredText(String colorName, String message) {
```

Comment:
very clear method name, but a bit long for all the times you had to type it. I wonder if simply `coloredText()` would have been equally clear?